components/Activity/Agenda/FileList.tsx

```diff
@@ -0,0 +1,22 @@
+import { TableCellAttachment } from 'mobx-lark';
+import { FC } from 'react';
+import { Badge, Col, Row } from 'react-bootstrap';
+
+import { i18n } from '../../../models/Base/Translation';
+
+const { t } = i18n;
+
+export const FileList: FC<{ fileInfo: TableCellAttachment[] }> = ({
+  fileInfo,
+}) => (
+  <Row as="section" className="justify-content-center" xs={1}>
+    <h2>{t('data_download')}</h2>
+    {fileInfo.map(({ id, name, attachmentToken }) => (
+      <Col key={id + ''}>
+        <a href={`/api/lark/file/${attachmentToken}`} download={name}>
+          <Badge bg="success">{name}</Badge>
+        </a>
```

Comment:
```suggestion
export const FileList: FC<{ data: TableCellAttachment[] }> = ({ data }) => (
  <Row as="section" className="justify-content-center" xs={1}>
    <h2>{t('data_download')}</h2>
    {data.map(({ id, name, mimeType, attachmentToken }) => (
      <Col key={id + ''}>
        <FilePreview type={mimeType} path={`/api/lark/file/${attachmentToken}`} />
        <Badge bg="success">{name}</Badge>
```
此处只演示**文件预览**，其它该改的还要改。